docs/UserGuide/Appendix/SQL-Reference.md

```diff
@@ -716,9 +716,27 @@ In this situation, it will throws an exception if * corresponds to multiple sens
 
 ```
 
+* Regexp Statement
+
+Regexp Statement only supports regular expressions with data type of text and Java standard library style when matching
```

Comment:
Regexp Statement only supports  regular expressions with Java standard library style on timeseries which is TEXT data type